### PR TITLE
[docs] typo fix

### PR DESCRIPTION
**Pull request by Armanio**
Mini typo fix 🤷‍♂️

### CODE REVIEW

File: docs/API.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ it('can handle commands synchronously', () => {
 you need return the command promise so it gets resolved properly as well as access the value when the promise got resolve:
 
 ```js
-it('handles commands as promises', ()  =>{
+it('handles commands as promises', ()  => {
```
Comment (christian-bromann) [inline docs/API.md]:
```suggestion
it('handles commands as promises', () => {
```

while we at it 😉

Comment (CrispusDH) [reply docs/API.md]: hm, I will check whether exists or not eslint rule for it :)

Comment (BorisOsipov) [reply docs/API.md]: It is docs. Linter doesn't process `.md` files

Comment (Armanio) [reply docs/API.md]: Fixed!

**Pull request merged**